nuclia/sdk/kbs.py

```diff
@@ -1,12 +1,13 @@
 from typing import Dict, Optional
 
 from nuclia import BASE
-from nuclia.config import retrieve
+from nuclia.config import KnowledgeBox, retrieve
 from nuclia.data import get_auth
-from nuclia.decorators import accounts
+from nuclia.decorators import account, accounts
 from nuclia.sdk.auth import NucliaAuth
 
-ADD_KB = BASE + "/api/v1/account/{account}/kbs"
+KBS_ENDPOINT = BASE + "/api/v1/account/{account}/kbs"
+KB_ENDPOINT = BASE + "/api/v1/account/{account}/kb/{slug}"
```

Comment:
if you want to use the KB slug you need to use `/kb/s/{slug}`: https://github.com/nuclia/nucliadb/blob/main/nucliadb/nucliadb/reader/api/v1/knowledgebox.py#L80

Reply:
Here we are on the global API, not on NucliaDB API, the kbid does not exist.